bindep.txt

```diff
@@ -1,6 +1,9 @@
-libxml2-devel [platform:rpm]
+openssl-devel [platform:rpm compile]
+gcc [platform:rpm compile]
+libcurl-devel [platform:rpm compile]
+libxml2-devel [platform:rpm compile]
 python3-pycurl [platform:rpm]
 python3-netaddr [platform:rpm]
 python3-jmespath [platform:rpm]
-python3-passlib [platform:rpm]
+python3-passlib [platform:rpm epel]
```

Comment:
I think we could change this to
```
python3-passlib [platform:fedora]
python3-passlib [platform:centos epel]
```

In case someone every build a fedora image.

Reply:
Good point. Done.